package.json

```diff
@@ -42,7 +42,7 @@
     "in-publish": "^2.0.0",
     "ink-docstrap": "^1.1.4",
     "isparta": "^4.0.0",
-    "jsdoc": "^3.4.0",
+    "jsdoc": "^3.5.5",
```

Comment:
If we're going to update jsdoc perhaps we should also update other dependencies. Let's use `yarn upgrade-interactive` or `npm-check` for this and make sure the tests still pass afterwards.

Additionally, I would bump the minimum Node version (in package.json `engines` and in `circle.yml`) to 6. We could even go to 8 but let's be conservative.